data/processors/poi.py

```diff
@@ -0,0 +1,27 @@
+from processors import merge
+
+
+def merge_poi(data):
+    """Merge POIs from `sources/21_pois.yaml` into the data"""
+    poi_data = merge.load_yaml("sources/21_pois.yaml")
+
+    for _id, poi in poi_data.items():
+        if _id in data:
+            raise ValueError(f"The id '{_id}' is already used, cannot use it for a new POI")
+
+        if poi["parent"] not in data:
+            raise ValueError(f"Parent '{poi['parent']}' of POI '{_id}' not found")
+
+        poi["type"] = "poi"
+
+        parent = data[poi["parent"]]
+        poi["parents"] = parent["parents"] + [parent["id"]]
+        del poi["parent"]
+
+        poi.setdefault("sources", {}).setdefault("base", []).append(
+            {
+                "name": "NavigaTUM",
+            },
+        )
```

Comment:
Your point here is probably, that we did some work and should be in the sources?
I would argue that if somebody added this while explicitly specifying the `source`, adding ourselves would not be "correct" (not a strong opinion)

```suggestion
        poi.setdefault("sources", {"base": [{"name": "NavigaTUM"}]})
```

Reply:
> Your point here is probably, that we did some work and should be in the sources?

I didn't pay enough attention to the code i copied over and adapted. You're right of course, it should be like this.